app/scripts/components/exploration/components/timeline/date-axis.tsx

```diff
@@ -75,6 +68,71 @@ function getTicks(scale: ScaleTime<number, number>) {
     .filter((v, i, a) => (width / a.length < 60 ? !(i % 2) : true));
 }
 
+/**
+ * Generates minor ticks for the date axis based on the major ticks and time density.
+ * Minor ticks are shown between major ticks and provide finer granularity on the timeline.
+
+ * @param {ScaleTime<number, number>} scale - The scale function for the timeline, mapping dates to pixel positions.
+ * @param {Date[]} majorTicks - The major tick points on the timeline (e.g., start of the month).
+ * @param {TimeDensity} timeDensity - The density of the timeline aka the level of detail (e.g., year, month, day).
+ * @returns {Date[]} - An array of minor tick dates.
+ */
+function getMinorTicks(scale: ScaleTime<number, number>, majorTicks: Date[], timeDensity: TimeDensity): Date[] {
+  if (timeDensity === TimeDensity.DAY) {
+    return [];
+  }
+
+  const [viewStart, viewEnd] = scale.domain();
+
+  // Number of segments to divide the interval between major ticks
+  // Set to 10 to for a balance between the level of detail, but also
+  // it offers a better alignment of the timeline playhead over the
+  // minor ticks.
+  const segments = 10;
+
+  // Ensure there are at least two major ticks to define an interval
+  if (majorTicks.length < 2) {
+    return [];
+  }
+
+  // Calculate the interval between minor ticks based on the first two major ticks
+  const minorTickInterval = (majorTicks[1].getTime() - majorTicks[0].getTime()) / segments;
+
+  // Initialize minorTicks as an array of Date objects
+  let minorTicks: Date[] = [];
+
+  // Add minor ticks before the first major tick (left hand side of
+  // the timeline) until reaching the view start
+  let tickTime = majorTicks[0].getTime() - minorTickInterval;
+
+  while (tickTime > viewStart.getTime()) {
```

Comment:
aha this should be done in three parts like this, such a detailed touch!

Reply:
yeah, I couldn't figure out any other way on how to show the minor vertical ticks on the edges (after the last major ticks), without them completely disappearing until another major tick pops up when a user swipes left / right.